src/routes/jira/events/jira-events-uninstall-post.test.ts

```diff
@@ -61,10 +61,10 @@ describe("Webhook: /events/uninstalled", () => {
 
 	it("Existing Installation", async () => {
 		when(jest.mocked(getJiraClient))
-			.calledWith(jiraHost, undefined, undefined, expect.anything())
```

Comment:
This is where the test wasn't working. `expect.anything()` of the 4th arg is not matching `req.log`.

In previous code, `req.log` is actually `undefined` inside `JiraEventsUninstallPost`  (although you set it up at `frontendApp`, but that's not being hit  coz `JiraEventsUninstallPost` was hit directly.

And because of `req.log` is undefined, `expect.anything()` will not match it. [expect.anything() matches anything but null or undefined](https://jestjs.io/docs/expect#expectanything)